app/assets/stylesheets/pages/settings.scss

```diff
@@ -92,4 +92,16 @@ $sidebar-item-highlight: #00c081;
     margin-top: 0;
     margin-bottom: 15px;
   }
-}
\ No newline at end of file
+}
+
+.ldap-conn-message-default {
+  color: black;
+}
+
+.ldap-conn-message-invalid {
+  color: red;
+}
+
+.ldap-conn-message-valid {
+  color: #04a36f;
```

Comment:
You can replace the hardcoded color with predefined vars. In this case `$brand-success`.

Reply:
Fixed now.